test/jdk/jdk/jfr/event/runtime/TestNativeLibraryLoadEvent.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2013, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
New test should have single copyright year, even if copied from existing test. Thanks.